apps/site/src/lib/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,35 @@
+.navMenuListWrapper {
+	// clip top only when mobile
+	@media (max-width: 768px) {
+		clip-path: inset(0px 0px -100px -100px);
+	}
+}
+
+.navMenuLink {
+	$underline-size: 2px;
+
+	font-size: 24px;
+	text-shadow: 0px 0px 20px #fffce2;
+
+	// needed for underline effect to work
+	padding-bottom: 0.5rem;
+
+	background-image: linear-gradient(white, white);
+	background-repeat: no-repeat;
+	background-position-y: 34px;
+	background-size: 0% $underline-size;
+	transition:
+		filter 0.1s ease,
```

Comment:
Nitpick: since we're not using the `filter` property any more, we don't need to include it in the transition.